event.go

```diff
@@ -0,0 +1,47 @@
+package instana
+
+type EventData struct {
+	Title    string `json:"title"`
+	Text     string `json:"text"`
+	Duration int    `json:"duration"`
+	Severity int    `json:"severity"`
+	Plugin   string `json:"plugin,omitempty"`
+	ID       string `json:"id,omitempty"`
+	Host     string `json:"host,omitempty"`
+}
+
+const (
+	ServicePlugin = "com.instana.forge.connection.http.logical.LogicalWebApp"
+	ServiceHost   = ""
+)
+
+func SendDefaultServiceEvent(title string, text string, severity int) {
+	SendServiceEvent(sensor.serviceName, title, text, severity)
+}
+
+func SendServiceEvent(service string, title string, text string, severity int) {
+	sendEvent(&EventData{
```

Comment:
This is agnostic.  Can send events at anytime regardless if process is announced or not.  We may want to test error handling eventually though (in the case of no host agent) and make sure things are handled graciously.